guides/common/modules/ref_dhcp-isc-settings.adoc

```diff
@@ -0,0 +1,43 @@
+[id="DHCP_ISC_Settings_{context}"]
+= dhcp_isc Settings
+
+The _dhcp_isc_ provider uses a combination of the ISC DHCP server OMAPI management interface and parsing of configuration and lease files.
+This requires it to be run either on the same host as the DHCP server or to have network filesystem access to these files.
```

Comment:
We have `dhcp_remote_isc` for the network access.
```suggestion
This requires it to be run either on the same host as the DHCP server.
```

Reply:
Thanks, applied!